x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/get_filter.ts

```diff
@@ -19,11 +19,11 @@ export const getFilter = ({
 
```

Comment:
I can see in this file this line: `// TODO (Jiawei): Use node builder instead of strings`
Maybe it's good place to fix it?
Or maybe we have separate ticket I do not know about.:-)

Reply:
Hmmm maybe you're right, we might as well do it or else it will never get done haha